composer.json

```diff
@@ -5,7 +5,7 @@
   },
   "require": {
     "firebase/php-jwt": "^3.0",
-    "shopware/connect-sdk": "dev-master"
+    "shopware/connect-sdk": "dev-CON-4609"
```

Comment:
this is for travis only and has to be changed before merge.
please don't approve until this is changed ;)